### PR TITLE
Cinema 101

### PR DESCRIPTION
**Pull request by mihalskiy**

### CODE REVIEW

File: src/layouts/components/layout-main.tsx [hunk 1]
```diff
@@ -18,9 +18,361 @@ export function LayoutMain({className = '', children = undefined}) {
...
+
+            <div id="news">
+                <div className="head">
+                    <h3>NEWS</h3>
+                    <p className="text-right"><a href="#">See all</a></p>
+                </div>
+
+                <div className="content">
+                    <p className="date">12.04.09</p>
+                    <h4>Disney's A Christmas Carol</h4>
```
Comment (odykyi) [inline src/layouts/components/layout-main.tsx]: not valid html
red arrow

File: src/layouts/components/layout-main.tsx [hunk 2]
```diff
@@ -18,9 +18,361 @@ export function LayoutMain({className = '', children = undefined}) {
...
+                    <p>&quot;Disney's A Christmas Carol,&quot; a multi-sensory thrill ride re-envisioned by Academy Award&reg;-winning filmmaker Robert Zemeckis, captures... </p>
```
Comment (odykyi) [inline src/layouts/components/layout-main.tsx]: not valid html
red arrow

File: src/layouts/components/layout-header.tsx [hunk 3]
```diff
@@ -11,7 +11,47 @@ export function LayoutHeader({className = '', children = undefined}) {
...
+                <ul>
+                    <li><a href="#">SHOW ALL</a></li>
+                    <li><a href="#">LATEST TRAILERS</a></li>
+                    <li><a href="#">TOP RATED</a></li>
+                    <li><a href="#">MOST COMMENTED</a></li>
+                </ul>
+                <div id="search">
+                    <form action="home_submit" method="get" accept-charset="utf-8">
+                        <label for="search-field">SEARCH</label>
+                        <input type="text" name="search field" value="Enter search here" id="search-field" title="Enter search here" className="blink search-field"  />
```
Comment (odykyi) [inline src/layouts/components/layout-header.tsx]: not need backspace

File: src/layouts/main-layout.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import './main-layout.css!';
+import './ie6.css!';
```
Comment (odykyi) [inline src/layouts/main-layout.tsx]: why you use `!` symbol

File: src/layouts/main-layout.tsx [hunk 5]
```diff
@@ -10,22 +11,17 @@ export class MainLayout extends React.Component<{}, {}> {
...
-            <LayoutTopNavLink href="/" isPrimary> Home </LayoutTopNavLink>
-            <LayoutTopNavLink href="/currency-converter"> Currency Converter </LayoutTopNavLink>
-            <LayoutTopNavLink href="/css-modules"> CSS Modules </LayoutTopNavLink>
-          </LayoutTopNav>
+
         </LayoutHeader>
 
         <LayoutMain>
-          {children}
+
```
Comment (odykyi) [inline src/layouts/main-layout.tsx]: do not remove `{children}`
its single page application
`children` will be change if you go to another route

File: src/layouts/components/layout-main.tsx [hunk 6]
```diff
@@ -31,7 +31,7 @@ export function LayoutMain({className = '', children = undefined}) {
 
                         <div className="movie-image">
 
-                            <a href="#"><span className="play"><span className="name">X-MAN</span></span><img src="../../../assets/../../../assets/images/movie1.jpg" alt="movie" /></a>
+                            <a href="#"><span className="play"><span className="name">X-MAN</span></span><img src="../../../assets/../../../assets/../../../assets/images/movie1.jpg" alt="movie" /></a>
```
Comment (odykyi) [inline src/layouts/components/layout-main.tsx]: its stupid path

Comment (odykyi) [reply src/layouts/components/layout-main.tsx]: its shit

`src="../../../assets/../../../assets/../../../assets/images/movie1.jpg"`

File: src/layouts/main-layout.tsx (hunk 5 above)
Comment (mihalskiy) [reply src/layouts/main-layout.tsx]: thanks for the comment now I will remember this all my life

File: src/layouts/main-layout.tsx (hunk 4 above)
Comment (mihalskiy) [reply src/layouts/main-layout.tsx]: these symbols require the syntax of the reaction

**Pull request merged**